src/bin/tf.rs

```diff
@@ -23,17 +23,17 @@ fn main() -> Result<(), Error> {
     Ok(())
 }
 
-#[derive(Debug, StructOpt)]
-#[structopt(name = "topfew")]
+#[derive(Debug, Parser)]
+#[clap(name = "topfew")]
 struct Options {
     /// Fields to use as part of the line's key
-    #[structopt(long, short)]
+    #[clap(long, short)]
     fields: FieldSpec,
     /// Top number of matches to show
-    #[structopt(long, short = "n", default_value = "10")]
+    #[clap(long, short = 'n', default_value = "10")]
```

Comment:
I think we can remove the values from the `short` arguments here, because clap will derive them for us.

Reply:
We can remove the ` = 'n'`, we can't remove the other one because the derived short would be `-r` not `-e` for `regex`